crates/miden-lib/src/testing/note.rs

```diff
@@ -69,11 +69,11 @@ impl NoteBuilder {
     /// Set the note's input to `inputs`.
     ///
     /// Note: This overwrite the inputs, the previous input values are discarded.
-    pub fn note_inputs(
+    pub fn note_storage_values(
```

Comment:
The docs need a "note inputs" -> "note storage" update.

Also, I would just call this `note_storage` because "values" is quite generic, so it doesn't make things much clearer.